courses/utils.py

```diff
@@ -0,0 +1,91 @@
+"""
+Utility functions for the courses app
+"""
+import re
+from opaque_keys.edx.keys import CourseKey
+from opaque_keys import InvalidKeyError
+
+NUMBER_SEASON_MAP = {
+    1: 'Spring',
+    2: 'Summer',
+    3: 'Fall'
+}
+
+
+def parse_edx_key(edx_course_key):
+    """
+    Attempts to parse a CourseRun's edx_course_key as an edX opaque key and return the portion
+    of the key that indicates the course's semester/year
+
+    Args:
+        edx_course_key (str): An edX course key (CourseRun.edx_course_key)
+
+    Returns:
+        The 'run' portion of a parsed CourseKey (opaque_keys.edx.keys.CourseKey#run),
```

Comment:
Can you add the return type like this: `str: The 'run' portion of a parsed CourseKey...`? These docstrings are parsed by PyCharm so it helps to be specific about syntax